src/gphotos_sync/authorize.py

```diff
@@ -104,7 +110,7 @@ def authorize(self):
                 "refresh_token": flow.credentials.refresh_token,
                 "token_type": "Bearer",
                 "scope": flow.credentials.scopes,
-                "expires_at": flow.credentials.expiry.timestamp(),
+                "expires_at": self.convert_to_local_timestamp(flow.credentials.expiry),
```

Comment:
```suggestion
                "expires_at": flow.credentials.expiry.replace(tzinfo=timezone.utc).timestamp()
```
+ `from datetime import timezone`

Reply:
This is a cleaner solution, thank you.